products/dns/terraform.yaml

```
@@ -26,6 +26,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           zone_name: "private-zone"
           network_1_name: "network-1"
           network_2_name: "network-2"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_managed_zone_private_peering"
```

```suggestion
        name: "dns_managed_zone_private_peering"
        min_version: 'beta'
```

Since this is using a beta feature, we should mark the example as beta. That's gonna make it require `google-beta` to be explicilty specified on each resource in the example; see https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/examples/region_autoscaler_beta.tf.erb